### ISSUE TITLE
ice aspect enchant - particle render crash

### ISSUE THREAD

**Issue by tonyoldfield** [SilentChaos512/SilentGems#173]
Playing the sprout modpack v0.91 which includes Silent's Gems v2.3.16-184.

Enchanted a sword with ice aspect II.
Using the sword in combat (7 times out of 10 as a guessitmation) now causes a crash to desktop.

<details>
<summary> 
Stack trace
</summary>
<p>
at net.minecraft.world.chunk.storage.ExtendedBlockStorage.func_76670_c(ExtendedBlockStorage.java:88)
	at net.minecraft.world.chunk.Chunk.func_177413_a(Chunk.java:643)
	at net.minecraft.world.World.func_175705_a(World.java:767)
	at net.minecraft.world.World.func_175626_b(World.java:818)
	at net.minecraft.client.particle.Particle.func_189214_a(SourceFile:309)
	at net.minecraft.client.particle.Particle.func_180434_a(SourceFile:171)
	at net.silentchaos512.gems.client.render.particle.EntityFXFreezing.func_180434_a(EntityFXFreezing.java:71)
	at net.minecraft.client.particle.ParticleManager.func_78874_a(ParticleManager.java:307)
	at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1355)
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1229)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1038)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1076)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
</p>
</details>
<p>
Full crash log available here https://pastebin.com/H2UQngte 
</p>

**Comment by SilentChaos512** [comment]
I tried the Sprout modpack and can't get it to crash. Have you added any extra mods?

**Comment by tonyoldfield** [comment]
No, it was still a locked 0.91 profile at that time.

Since then I have unlocked the profile and been fiddling. Turns out the crash only happens when the Dynamic Surroundings mod is enabled.

That mod has been crashing intermittently, causing short hangs in my client that take a moment to recover from, so it was actually the first thing I turned off after unlocking the profile, just as a quality of life improvement - and also fixed this issue!

I suppose your millage may vary with reproducing the issue depending on how stable dynamic surroundings is running for you.
